crates/ruff_cli/src/args.rs

```diff
@@ -351,6 +351,9 @@ pub struct CheckCommand {
         conflicts_with = "watch",
     )]
     pub show_settings: bool,
+    /// List of mappings from file extension to language (one of ["python", "ipynb", "pyi"]).
+    #[arg(long, value_delimiter = ',', hide = true)]
+    pub extension_override: Option<Vec<ExtensionPair>>,
```

Comment:
I'm somewhat partial to calling this `extension` rather than `extension-override`.

Reply:
Would you prefer it changed everywhere or just here and `CheckArguments`?
I used `extension_override` rather than `extension` just because it seemed more explicit to me.  Why do prefer `extension`?

Reply:
renamed